shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/TripStopTime.kt

```diff
@@ -0,0 +1,16 @@
+package com.mbta.tid.mbta_app.model
+
+import kotlinx.datetime.Instant
+
+interface TripStopTime : Comparable<TripStopTime> {
+    val arrivalTime: Instant?
+    val departureTime: Instant?
```

Comment:
This is an abstraction that I thought of months ago but haven't yet had a compelling reason to add. If I'd been more eager to implement it, I would still remember more places where we could take advantage of it; as it stands, there's only one existing piece of code that I've changed to use this abstraction.